src/main/groovy/io/seqera/wave/service/blob/impl/KubeTransferStrategy.groovy

```diff
@@ -48,17 +48,19 @@ class KubeTransferStrategy implements TransferStrategy {
 
     @Override
     BlobCacheInfo transfer(BlobCacheInfo info, List<String> command) {
-        final podName = podName(info)
-        final pod = k8sService.transferContainer(podName, blobConfig.s5Image, command, blobConfig)
+        final podName = getName(info, "pod")
+        final jobName = getName(info, "job")
+        final job = k8sService.transferJob(jobName, podName, blobConfig.s5Image, command, blobConfig)
+        final pod = k8sService.getPod(podName)
         final terminated = k8sService.waitPod(pod, blobConfig.transferTimeout.toMillis())
         final stdout = k8sService.logsPod(podName)
         return terminated
                 ? info.completed(terminated.exitCode, stdout)
                 : info.failed(stdout)
     }
 
-    protected String podName(BlobCacheInfo info) {
-        return 'transfer-' + Hashing
+    protected static String getName(BlobCacheInfo info, String type) {
+        return "transfer-$type-" + Hashing
```

Comment:
Is the type really needed?

Reply:
not needed, I have checked and found that job and pod can have the same name
I will make the changes